check_empty_zones.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+import boto3
+
+
+def get_aws_zones():
+    route53 = boto3.client("route53")
```

Comment:
I'd recommend instantiating the Route53Client at the top level and passing it through to these functions 🤝 

Mainly because it shows at a high level what the external dependencies the script requires - and also what dependencies each functions require 🥂

Can imagine reading this function `get_aws_zones()` and thinking it could be reading from an S3Bucket, using a local variable or reading from a local file - but `get_aws_zones(route53Client)` is a bit clearer where the information is coming from 👀 

Also makes testing a bit easier 🧪